docs/getting-started.md

```diff
@@ -161,9 +164,9 @@ Content-Length: 0
 
 ### Issuing a Certificate
 
-Backends connect via gRPC secured with mTLS. For this we will issue a certificate from certs service.
+CVMs connect via gRPC secured with mTLS. For this we will issue a certificate from certs service.
```

Comment:
This section only applied for backends.
Certs are issued on CVM creation, the certs page is optional and is used when the user needs to issue new certs for the cvm.  The new certs here are to be manually sent into the cvm if they are to be used for connection between agent and CVMs service.

Reply:
certs can be issued by the user but this is not necessary or required for getting started.

Reply:
certs can be issued by the user but this is not necessary or required for getting started.